coderadar-server/coderadar-core/src/main/java/org/wickedsource/coderadar/security/CoderadarSecurityConfiguration.java

```diff
@@ -51,6 +53,12 @@ protected void configure(AuthenticationManagerBuilder auth) throws Exception {
     auth.userDetailsService(userDetailsService).passwordEncoder(getPasswordEncoder());
   }
 
+  @Bean(name = BeanIds.AUTHENTICATION_MANAGER)
```

Comment:
why did you introduce this bean?

Reply:
An AuthenticationManager Bean is required, but is no longer automatically created, so I created it myself. 
https://github.com/spring-projects/spring-boot/issues/11136 According to this conversation, the bean must be created manually when a custom `UserDetailsService` is present

Reply:
OK